arangod/Graph/TraverserDocumentCache.cpp

```diff
@@ -149,8 +149,8 @@ void TraverserDocumentCache::insertDocument(
           cache::CachedValue::construct(key, keySize, resVal, resValSize));
 
       if (value) {
-        bool success = _cache->insert(value.get());
-        if (!success) {
+        auto result = _cache->insert(value.get());
+        if (!result.ok()) {
           LOG_TOPIC(DEBUG, Logger::GRAPHS) << "Insert document into cache failed";
```

Comment:
Yes, leave it as it is...
I was confused because I thought the commit belonged to the general cache. There the log topic should be `Logger::CACHE`.
But as it is the `TraverserCache`, it can stay as it is. Sorry again for the confusion!